handlers/challenge_handler.py

```diff
@@ -500,7 +508,14 @@ def init(self, slack_wrapper):
                     ctf.add_challenge(challenge)
 
         # Create the database accordingly
-        pickle.dump(database, open(self.DB, "wb+"))
+        pickle.dump(database, open(ChallengeHandler.DB, "wb+"))
+
+    def init(self, slack_wrapper):
+        """
```

Comment:
Minor: Put into one line.

Reply:
We also probably don't need doc strings for things as obvious as this.

Reply:
It's a remnant from the previous code.